Documentation/api-reference.md

```diff
@@ -69,14 +69,21 @@ setLogLevel: function(level as integer) as void
 
 ```brightscript
 ADB_CONSTANTS = AdobeSDKConstants()
-m.adobeEdgeSdk.setLogLevel(ADB_CONSTANTS.LOG_LEVEL.VERBOSE)
+m.aepSdk.setLogLevel(ADB_CONSTANTS.LOG_LEVEL.VERBOSE)
 ```
 
 ---
 
 ### updateConfiguration
 
-> It is better to call this function before calling any other public APIs. For example, if calling sendEvent() without a valid configuration in the SDK, the Edge requests will be queued locally and sent out later with the valid configuration.
+> This API should be called before any other public APIs. Public APIs need valid configuration to process the data and make the network call to Adobe Experience Edge Network. All the hits will be queued if no valid configuration is found.
```

Comment:
Because we have supported the request queueing, and customers may spend some time loading configuration (e.g. from the local file or the remote server). Can we reword a bit and avoid using `should`.

Reply:
I wanted to assert the ideal workflow. Let me reword this.